discord/commands/commands.py

```diff
@@ -631,7 +631,7 @@ async def _invoke(self, ctx: ApplicationContext) -> None:
             await self.callback(ctx, **kwargs)
 
     async def invoke_autocomplete_callback(self, ctx: AutocompleteContext):
-        values = { i.name: i.default for i in self.options }
+        values = {i.name: i.default for i in self.options}
```

Comment:
Inconsistent formatting, should probably look like this:
 https://github.com/Pycord-Development/pycord/blob/5b4f30f5739724cf3c473b66a56c668f705ad210/discord/commands/commands.py#L785

Reply:
This is not a pull request change?

Reply:
> This is not a pull request change?

I misread the code, mb